src/proxy-ui-api/frontend/src/views/Service/Parameters/ServiceParameters.vue

```diff
@@ -0,0 +1,488 @@
+          url_all: this.url_all,
+          ssl_auth_all: this.ssl_auth_all,
+        })
+        .then((res) => {
+          this.$bus.$emit('saveService', res.data);
+          this.$bus.$emit('show-success', 'Service saved');
+        })
+        .catch((error) => {
+          this.$bus.$emit('show-error', error.message);
+        });
+    },
+
+    setTouched(): void {
+      this.touched = true;
+    },
+
+    fetchData(serviceId: string): void {
+
+      api
+        .get(`/services/${serviceId}/access-rights`)
+        .then((res) => {
+          this.accessRightsSubjects = res.data;
+        })
+        .catch((error) => {
+          this.$bus.$emit('show-error', error.message);
+        });
+    },
+
+    showAddSubjectsDialog(): void {
+      this.addSubjectsDialogVisible = true;
+    },
+
+    doAddSubjects(selected: any[]): void {
+      this.addSubjectsDialogVisible = false;
+
+      api
+        .post(`/services/${this.serviceId}/access-rights`, {
+          items: selected,
+        })
+        .then((res) => {
+          this.fetchData(this.serviceId);
+        })
+        .catch((error) => {
+          this.$bus.$emit('show-error', error.message);
+        });
+    },
+
+    closeAccessRightsDialog(): void {
+      this.addSubjectsDialogVisible = false;
+    },
+
+    removeAllSubjects(): void {
+      this.confirmAllSubjects = true;
+    },
+
+    doRemoveAllSubjects(): void {
+      const subjects: any = [];
+      this.accessRightsSubjects.forEach((subject: any) => {
+        subjects.push({
+          id: subject.subject.id,
+          subject_type: subject.subject.subject_type,
+        });
+      });
+
+      this.removeArrayOfSubjects(subjects);
+      this.confirmAllSubjects = false;
+    },
+
+    removeSubject(member: any): void {
+      this.confirmMember = true;
+      this.selectedMember = member;
+    },
+    doRemoveSubject() {
+      const subject: ServiceClient = this.selectedMember as ServiceClient;
+
+      if (subject && subject.subject.id) {
+        this.removeArrayOfSubjects([
+          {
+            id: subject.subject.id,
+            subject_type: subject.subject.subject_type,
+          },
+        ]);
+      }
+
+      this.confirmMember = false;
+      this.selectedMember = undefined;
+    },
+
+    removeArrayOfSubjects(subjects: any) {
+      api
+        .post(`/services/${this.serviceId}/access-rights/delete`, {
+          items: subjects,
+        })
+        .catch((error) => {
+          this.$bus.$emit('show-error', error.message);
+        })
+        .finally(() => {
+          this.fetchData(this.serviceId);
+        });
+    },
+    close() {
+      this.$router.push({name: RouteName.SubsystemServices, params: {id: this.clientId}});
+    },
+  },
+  watch: {
+    isHttps(val) {
+      // If user edits http to https --> change "ssl auth" to true
+      if (val === true) {
+        this.service.ssl_auth = true;
+      }
+    },
+  },
+  created() {
```

Comment:
If this created is not used it can be deleted.

Reply:
This haven't been changed. Don't actually know what is this timeout variable for but it hasn't changed in this task (just moved to another file)